src/meetings/meetings.service.ts

```diff
@@ -127,7 +150,64 @@ export class MeetingsService {
         attendees: true,
       },
     });
+    this._meetings.next({
+      type: 'cancelled',
+      id: meeting.id,
+    });
     this.eventEmitter.emit('meeting.delete', meeting);
     return meeting;
   }
+
+  @OnEvent('meeting.start')
```

Comment:
For all the @OnEvent, and in the scheduler.service, and everytimes with use eventEmitter.emit, I think it would be better to create an enum called MeetingEvent and here, write MeetingEvent.Start instead of just a string. 

I don't know if it is a good convention with nestJs event emiter or not because I have never used it, but I think it could be better :D 

WDYT ?

Reply:
Yes, this might help to avoid typos.

Reply:
Created an issue for this : #77